src/settings/Validation/engine/handlers.test.js

```diff
@@ -0,0 +1,1481 @@
+      it('should return false', () => {
+        expect(isInIntervalOrNull(5, 10, 11)).toBe(false);
+      });
+    });
+
+    describe('when passed value is less than min value', () => {
+      it('should return false', () => {
+        expect(isInIntervalOrNull(5, 10, 4)).toBe(false);
+      });
+    });
+  });
+
+  describe('isNotEmptyEditor method', () => {
+    describe('when passed value has content after removing tags', () => {
+      it('should return true', () => {
+        expect(isNotEmptyEditor('<h1>hello</h1><b>world</b>')).toBe(true);
+      });
+    });
+
+    describe('when passed value has space after removing tags', () => {
+      it('should return true', () => {
+        expect(isNotEmptyEditor(' <h1></h1><b></b>')).toBe(true);
+      });
+    });
+
+    describe('when passed value has no content after removing tags', () => {
+      it('should return false', () => {
+        expect(isNotEmptyEditor('<h1></h1><b></b>')).toBe(false);
+      });
+    });
+
+    describe('when value is not passed', () => {
+      it('should return false', () => {
+        expect(isNotEmptyEditor()).toBe(false);
+      });
+    });
+  });
+
+  describe('isSelected method', () => {
+    describe('when passed value is true', () => {
+      it('should return true', () => {
+        expect(isSelected(true)).toBe(true);
+      });
+    });
+
+    describe('when passed value is false', () => {
+      it('should return true', () => {
+        expect(isSelected(false)).toBe(true);
+      });
+    });
+
+    describe('when value is not passed', () => {
+      it('should return false', () => {
+        expect(isSelected()).toBe(false);
+      });
+    });
+  });
+
+  describe('isGreaterOrEqualThanPassedField method', () => {
+    describe('when passed value is greater than passed field', () => {
+      const fieldToCompare = 'testProperty';
+      const model = {
+        [fieldToCompare]: 10,
+      };
+
+      it('should return true', () => {
+        expect(isGreaterOrEqualThanPassedField(fieldToCompare, 20, model)).toBe(true);
+      });
+    });
+
+    describe('when passed value is equal to passed field', () => {
+      const fieldToCompare = 'testProperty';
+      const model = {
+        [fieldToCompare]: 10,
+      };
+
+      it('should return true', () => {
+        expect(isGreaterOrEqualThanPassedField(fieldToCompare, 10, model)).toBe(true);
+      });
+    });
+
+    describe('when passed value is less than passed field', () => {
+      const fieldToCompare = 'testProperty';
+      const model = {
+        [fieldToCompare]: 10,
+      };
+
+      it('should return false', () => {
+        expect(isGreaterOrEqualThanPassedField(fieldToCompare, 9, model)).toBe(false);
+      });
+    });
+  });
+
+  describe('isMaximumFineValueValid method', () => {
+    describe('when passed value is more than zero and valueToCompare is more than zero', () => {
+      const fieldToCompare = 'testProperty';
```

Comment:
I would suggest moving that variable to the top. I see that you create variables with the same name in a few places.

Reply:
done. thanks.